src/components/Footer/FooterLinks/index.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import boleto from './icons/boleto.svg';
+import elo from './icons/elo.svg';
+import mastercard from './icons/mastercard.svg';
+import paypal from './icons/paypal.svg';
+import visa from './icons/visa.svg';
+import telephone from './icons/telephone.svg';
+import email from './icons/email.svg';
+import {
+    FooterContainer,
+    FooterCategories,
+    CategoryLinks,
+    SupportInfo,
+    FooterImages,
+} from './styles';
+
+const FooterLinks = [
+    {
+        title: 'Categories',
+        links: [
+            {
+                title: 'Peças',
+                path: 'xxx',
+            },
+            {
+                title: 'Serviços',
+                path: 'xxx',
+            },
+            {
+                title: 'Manutenção',
+                path: 'xxx',
+            },
+            {
+                title: 'Sistema de Trocas',
+                path: 'xxx',
+            },
+        ],
+    },
+    {
+        title: 'Institucional',
+        links: [
+            {
+                title: 'Quem Somos',
+                path: 'xxx',
+            },
+            {
+                title: 'Nosso Historia',
+                path: 'xxx',
+            },
+            {
+                title: 'Trabalhe Conosco',
+                path: 'xxx',
+            },
+        ],
+    },
+    {
+        title: 'Ajuda',
+        links: [
+            {
+                title: 'Política de Privacidade',
+                path: 'xxx',
+            },
+            {
+                title: 'Política de Troca',
+                path: 'xxx',
+            },
+            {
+                title: 'Termos e Condições',
+                path: 'xxx',
+            },
+            {
+                title: 'FAQ',
+                path: 'xxx',
+            },
+            {
+                title: 'Fale Conosco',
+                path: 'xxx',
+            },
+        ],
+    },
+    {
+        title: 'Suporte',
+        links: [
+            {
+                number: '080000000',
+                email: 'ajuda@scania.com.br',
```

Comment:
Use another name here, this store is not the `Scania` store. Use the name of the new logo.